config/tcs_config.toml

```diff
@@ -49,7 +49,7 @@ num_of_enclaves = "1"
 ias_url = "https://api.trustedservices.intel.com/sgx/dev"
 
 # Proxy for https. Change to your corporate proxy or comment out below line for direct Internet connections.
-https_proxy = "http://proxy-iind.intel.com:912/"
+#https_proxy = "http://your-proxy:your-port/"
 
 # IAS api key is subscription key used for authentication of requests submitted to IAS server.
 # this can be obtained by subscribing in the portal https://api.portal.trustedservices.intel.com.
```

Comment:
Is this a valid ias_api_key? Can we replace it with the dummy value here?

Reply:
I do not know if the example key is valid.  I replaced it with a dummy value.